### ISSUE TITLE
Status Document's Event.type not valid

### ISSUE THREAD

**Issue by camill-a** [readium/readium-lcp-server#145]
The returned value of the Event's Type is `"1"`when it should be a string ["register", "renew", "return"]

```
/// License Document from  https://rcs-test.edrlab.org/10013/purchases/89/status
 [...] },
    "events":[
              {
              "name":"DRMInside/iOS/(null)",
              "timestamp":"2017-08-30T08:38:01.114560919Z",
              "type":"1",
              "id":"32857C18-5CC4-478D-A0F6-A8E90D24D89E"
              }
              ]
}
```

**Comment by danielweck** [comment]
Agreed:
https://readium.github.io/readium-lsd-specification/#27-events

**Comment by danielweck** [comment]
See:
https://github.com/readium/readium-lcp-server/blob/ebac9cd34c852136f53db73e0ae13c243eed9790/lsdserver/api/license_status.go#L827-L836

**Comment by danielweck** [comment]
The LSD JSON is actually created here:
https://github.com/readium/readium-lcp-server/blob/ebac9cd34c852136f53db73e0ae13c243eed9790/lsdserver/api/license_status.go#L767-L783

**Comment by llemeurfr** [comment]
corrected in the develop branch, 49fe992885d864f452a93f626c4a36b452587512
